README.md

```diff
@@ -1,70 +1,174 @@
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-## Available Scripts
+<!-- PROJECT DESCRIPTION -->
 
-In the project directory, you can run:
+# 📖 [Crypto Trend] <a name="about-project"></a>
 
-### `npm start`
+**[Description]**
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+The application was created with React and Redux, using the coinranking API.
+[Crypto Trend] is the ultimate online platform for crypto trends, built with HTML, CSS, React, and Redux. Stay ahead of the curve, explore new opportunities, and make informed decisions in the exciting world of cryptocurrencies
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## 🛠 Built With <a name="built-with"></a>
 
-### `npm test`
+### Tech Stack <a name="tech-stack"></a>
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+HTML, CSS & JAVASCRIPT
 
-### `npm run build`
+GitHub & Visual Studio Code
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+React and Redux
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+API Yelp fuction
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+<!-- Features -->
 
-### `npm run eject`
+### Key Features <a name="key-features"></a>
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+- Use CreateAsyncThunk in Redux.
+- Fetch datas from a database with Redux
+- Conditional rendering in React
+- Filtering out selected items to another screen
+- Single Page App
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+<!-- LIVE DEMO -->
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+## 🚀 Live Demo <a name="live-demo"></a>
 
-## Learn More
+- [Live Demo Link](https://bit-coin-app.onrender.com)
+  [Video Presentation](https://www.loom.com/share/1cf7fe824cbf451b9675fb0a658afc71)
```

Comment:
- [ ] As required by the previous reviewer, I kindly suggest that you please talk about a piece of code that you're proud of in the video presentation.